partitioned-table.md

```diff
@@ -579,7 +579,19 @@ You can see that the inserted record `(NULL, 'mothra')` falls into the same part
 
 ## Partition management
 
-You can add, drop, merge, split, redefine partitions by using `ALTER TABLE` statements.
+For `LIST` and `RANGE` partitioned tables, you can add and drop partitions using the `ALTER TABLE <table name> ADD PARTITION (<partition specification>)` or `ALTER TABLE <table name> DROP PARTITION <list of partitions>` statement.
+
+For `HASH` partitioned tables, `COALESCE PARTITION` and `ADD PARTITION` is not yet supported.
+
+`REORGANIZE PARTITION` is not yet supported.
+
+> **Warning:**
+>
+> - `EXCHANGE PARTITION` is an experimental feature. It is not recommended to use it in a production environment. To enable it, set the `tidb_enable_exchange_partition` system variable to `ON`.
+> 
+> - It works by swapping a partition and a non-partitioned table, similar to how renaming a  table like `RENAME TABLE t1 TO t1_tmp, t2 TO t1, t1_tmp TO t2` works.
```

Comment:
What does "it" refer to here?

Reply:
```suggestion
> - `EXCHANGE PARTITION` works by swapping a partition and a non-partitioned table, similar to how renaming a  table like `RENAME TABLE t1 TO t1_tmp, t2 TO t1, t1_tmp TO t2` works.
```

Reply:
Thanks for the suggestion, I have updated it.